src/utils/filteredImages.ts

```
@@ -0,0 +1,261 @@
+        images.sort((a, b) => {
+          const dateA = a.created_at ? new Date(a.created_at).getTime() : 0;
+          const dateB = b.created_at ? new Date(b.created_at).getTime() : 0;
+          const result = dateA - dateB;
+          return state.sortDirection === "descending" ? -result : result;
+        });
+        break;
+      case "lastCategorized":
+        // Use cached snapshot if suppressCategoryRefilter is active (defer refiltering)
+        const imageCategoriesForSorting = state.suppressCategoryRefilter && state.cachedImageCategoriesForRefilter
+          ? state.cachedImageCategoriesForRefilter
+          : state.imageCategories;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)